packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -221,9 +221,10 @@ const argv = yargs
       describe: 'More verbose logging',
       type: 'boolean',
     },
-    'noWatchman': {
-      describe: 'Do not use watchman when not in watch mode',
+    'watchman': {
+      describe: 'Use watchman when not in watch mode',
```

Comment:
Watchman will always be used in watch mode. Watchman will also still be used when not in watch mode unless `--watchman=false` is passed or we detect watchman is not installed.